src/Compilers/Test/Core/Compilation/ControlFlowGraphVerifier.cs

```diff
@@ -892,6 +901,54 @@ bool isObjectInitializerInitializedObjectTarget(IFlowCaptureReferenceOperation r
                 } && left == referenceSyntax;
             }
 
+            bool isInterpolatedStringArgumentCapture(IFlowCaptureReferenceOperation reference)
+            {
+                if (reference.Language != LanguageNames.CSharp)
+                {
+                    return false;
+                }
+
+#pragma warning disable IDE0055 // Fix formatting
+                return reference is
+                       {
+                           Parent: IArgumentOperation
+                           {
+                               Parent: IObjectCreationOperation
+                               {
+                                   Parent: IFlowCaptureOperation,
+                                   Constructor.ContainingType: INamedTypeSymbol ctorContainingType,
+                                   Arguments: { Length: >= 3 } arguments,
+                                   Syntax: CSharp.Syntax.InterpolatedStringExpressionSyntax or CSharp.Syntax.BinaryExpressionSyntax
```

Comment:
>CSharp.Syntax.BinaryExpressionSyntax [](http://example.com/codeflow?start=95&length=36)

Should we make this stronger and actuall check that there is an interpolation addition?

Reply:
We could, but I don't think it's worth it. I don't know of any other way in C# to turn a binary expression syntax into an `IObjectCreationOperation` node, and the extra complexity from walking the tree (through any parenthesized/suppressed nodes) isn't worth the small extra verification.